pages/coming-soon.tsx

```diff
@@ -0,0 +1,5 @@
+const ComingSoon = () => {
```

Comment:
is this file used anywhere?

Reply:
I made it as a placeholder for the navbar, but since each link is keyed on the url, it would lead to conflicts. Will add a redirect separately for each page.